README.md

```diff
@@ -62,6 +62,15 @@ or you can clone the repository locally via
 ```python
 git clone git@github.com:fairlearn/fairlearn.git
 ```
+If you clone from git and wish to use the Fairness dashboard, then you will need to [install Yarn](https://yarnpkg.com/lang/en/docs/install), and then do the following:
```

Comment:
Maybe make clear that this is for using a local install, that you can always clone and tehn just pip install fairlearn without this? Unless that's too pedantic.

Reply:
If people cloned and did the pip install, which would actually run? Or did you mean do a `pip install -e .` ?

Reply:
I think we agreed that this was clear enough. It's likely that only developers would be doing the clone.